southpark-search/Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN python -c "from transformers import DistilBertModel, DistilBertTokenizer; x=
 COPY . /
 
 RUN bash get_data.sh ./data && \
-    python app.restful.py dryrun
+    python app.py -t dryrun
 
-ENTRYPOINT ["python", "app.restful.py"]
+ENTRYPOINT ["python", "app.py", "-t"]
```

Comment:
in which mode will it run by default? Won't it fail if no mode is given?

Reply:
When we build the docker image, we use the builtin.Dockerfile, this Dockerfile I just kept how it used to be

Reply:
It it is not used, let's remove this file. I think this dockerfile would not work. In the builtin.dockerfile it does `ENTRYPOINT ["python", "app.restful.py", "search"]`. So either we remove it or we add `search` to the options.

Reply:
Dockerfile removed and builtin.Dockerfile renamed to Dockerfile